platform/analysis-api/src/com/intellij/codeInsight/completion/CompletionSorter.java

```diff
@@ -22,10 +22,19 @@
  * @author peter
  */
 public abstract class CompletionSorter {
+  /**
```

Comment:
Could you please move the whole commit with documentation to a separate PR? So, IntelliJ team could review it separately

Reply:
Sure, sent https://github.com/JetBrains/intellij-community/pull/1677